storage/src/main/scala/io/casperlabs/storage/dag/SQLiteDagStorage.scala

```diff
@@ -253,33 +253,35 @@ class SQLiteDagStorage[F[_]: Sync](
 
   override def topoSortValidator(
       validator: Validator,
-      blocksNum: Int,
+      sliceDepth: Int,
       endBlockNumber: Long
-  ) = {
-    val subQuery = fr"""SELECT validator_block_seq_num, """ ++ blockInfoCols() ++ fr"""
-          FROM block_metadata
-          WHERE validator_block_seq_num<=$endBlockNumber AND validator=$validator
-          ORDER BY validator_block_seq_num DESC
-          LIMIT $blocksNum"""
-    (fr"SELECT * FROM (" ++ subQuery ++ fr") ORDER BY validator_block_seq_num ASC")
+  ) =
+    (fr"""SELECT validator_block_seq_num, """ ++ blockInfoCols() ++
+      fr"""
+          |FROM block_metadata a
+          |WHERE validator+$validator AND
```

Comment:
```suggestion
          |WHERE validator=$validator AND
```
do we have no tests for that?

Reply:
Added property test, thanks! I would probably never do that without your comment